README.md

```diff
@@ -75,7 +90,7 @@ device by putting the device into pairing mode and turning on a device id, light
 
 The device ids don't have to be unique, but it's advisable.
 
-Lighting2
+HomeEasy (EU)
```

Comment:
Not sure this change is necessary (or at least, it should be Lightin2/HomeEasy (EU) and perhaps the underline below should be extended.